package.json

```diff
@@ -64,13 +77,15 @@
   "devDependencies": {
     "babel-runtime": "5.8.29",
     "babelify": "6.4.0",
-    "browserify": "6.1.0",
+    "browserify": "10.0.0",
+    "browserify-derequire": "^0.9.4",
```

Comment:
I think that's a good idea. So you would like me to peg `browserify-derequire` at 0.9.4?